Items/EqpLunar/Lantern.cs

```diff
@@ -66,7 +66,23 @@ public class Lantern : Equipment<Lantern> {
             }
             return true;
         }
-    }
+
+		private string FormatNewLangPickup()
+        {
+			string desc = "Drop a lantern that fears";
+			if (damage > 0f) desc += " and damages";
+			desc += $" enemies for {duration:N0} seconds.";
+			return desc;
+        }
+
+		private string FormatNewLangDesc()
+		{
+			string desc = $"Sets a {range:N0}m, {duration:N0}-second AoE which <style=cIsUtility>fears enemies</style>";
```

Comment:
"###unit" is never grammatically correct. "### unit" usually is, but "###-nonAbbreviatedUnit" is preferred in some cases like the above.

Reply:
Noted.